static/flow/views/data-set-view.js

```diff
@@ -488,19 +488,14 @@ var DataSetView = function(options) {
         var dataPairs = base.m_plotHandler.plotter.dataPairs;
         if (dataPairs.length && dataPairs[0].xData.data.length) {
 
-            var diagram = base.m_program;
-
             // set collection attributes based on current input blocks
```

Comment:
Looks like this comment is out-of-date now

Reply:
Yup, it's based on sequence data, not the blocks stored in the program.  I pushed a fix.